frontend/src/components/About.vue

```diff
@@ -17,35 +22,56 @@
 
           <p>Metabolic Atlas would not be possible without the following excellent Elixir resources: <a href="https://proteinatlas.org" target="_blank">The Human Protein Atlas</a>, <a href="https://www.ensembl.org/" target="_blank">Ensembl</a>, <a href="https://www.uniprot.org" target="_blank">UniProt</a>, <a href="https://europepmc.org" target="_blank">Europe PMC</a>, <a href="https://identifiers.org" target="_blank">Identifiers.org</a>, <a href="https://www.ebi.ac.uk/chebi" target="_blank">ChEBi</a> and other essential sources of information: <a href="http://www.genome.jp/kegg/" target="_blank">KEGG: Kyoto Encyclopedia of Genes and Genomes</a>,  <a href="http://www.hmdb.ca/" target="_blank">The Human Metabolome Database</a>, <a href="https://www.metanetx.org" target="_blank">MetaNetX</a>, <a href="http://bigg.ucsd.edu" target="_blank">BiGG</a>, <a href="https://www.ncbi.nlm.nih.gov" target="_blank">NCBI</a>.
           </p>
         </div>
         <div class="column is-half has-text-justified content">
           <div class="box is-hidden-mobile">
             <p class="subtitle has-text-weight-bold is-size-4 has-text-primary has-text-centered">Citing Metabolic Atlas</p>
-            <a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482 </a>
+            <p><a href="https://doi.org/10.1073/pnas.2102344118" target="_blank">
+              Wang, H., et al, 2021. <i>Genome-scale metabolic network reconstruction of model animals as a platform for translational research</i>. PNAS 118, e2102344118
+            </a></p>
+            <p><a href="https://doi.org/10.1126/scisignal.aaz1482" target="_blank">
+              Robinson, J., et al, 2020. <i>An atlas of human metabolism</i>. Sci. Signal. 13, eaaz1482
+            </a></p>
           </div>
           <img width="100%" alt="The SysBio group 2019" src="../assets/group_19.jpg">
-          <center>The SysBio group 2019</center>
+          <center><a href="https://sysbio.se" target="_blank">The SysBio group 2019</a></center>
           <br><br>
-          <h4 class="title is-4">Scientific Advisors</h4>
-          <nav id="advisors" class="level">
-            <div v-for="advisor in advisors" :key="advisor.name" class="level-left">
-              <a :href="advisor.url" target="_blank">
-                <span class="level-item">
+          <h4 class="title is-4">Contact</h4>
+          <p>For any questions, comments or suggestions please email us at <a href="mailto:">contact [at] metabolicatlas [dot] org</a>.</p>
```

Comment:
Since we are using Cloudflare now, we could start including the email addresses in the mailto link. Not a big difference either way, just wanted to mention it if we would be interested in the future. https://support.cloudflare.com/hc/en-us/articles/200170016-What-is-Email-Address-Obfuscation-

Reply:
Great idea! This is now achieved through 8d70a05.